CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
```

Comment:
Oh shit: it adds this line every time, and not just for the first generation. I removed it, but if it keeps adding it, then let's leave it there.